README.md

```diff
@@ -60,9 +60,15 @@ SOLUÇÃO
 * A solução poderá ser desenvolvida em qualquer linguagem da sua escolha;
 * Seja criativo;
 * Uma outra pessoa estará lendo o que você fez, portanto, lembre-se de buscar trazer a semântica do problema no seu código;
-* Crie testes;
 * Não é necessário utilizar nenhum framework, procure utilizar os recursos da linguagem;
 
+GOSTARÍAMOS DE VER
```

Comment:
Guys, do you think we should state clearer what we want (must have) from what we'd like to see (nice to have)?
I really think *Tests* are not a Nice to Have.